msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenSilentSupplier.java

```diff
@@ -84,29 +66,7 @@ AuthenticationResult execute() throws Exception {
                         requestAuthority = Authority.createAuthority(new URL(requestAuthority.authority().replace(requestAuthority.host(),
                                 res.account().environment())));
                     }
```

Comment:
nit: as per comment above, control flow using exception is not great.

This is for line 76 (for some reason GH refuses to let me comment on that line...)

https://stackoverflow.com/questions/729379/why-not-use-exceptions-as-regular-flow-of-control